src/NServiceBus.AcceptanceTesting/NServiceBus.AcceptanceTesting.csproj

```diff
@@ -5,7 +5,8 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\NServiceBus.snk</AssemblyOriginatorKeyFile>
     <OutputPath>..\..\binaries\</OutputPath>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>	
+    <GeneratePackageOnBuild Condition="'$(Configuration)' == 'Release'">true</GeneratePackageOnBuild>
```

Comment:
Why has this been moved into each project? It should just be uncommented from the targets file.

Reply:
Also, why the condition? I saw that @timbussmann said something about it slowing things down? That's not been my experience.

Reply:
I haven't seen it slow things down either, but it was a requested change. It was moved into each file because, apparently, that is how all the downstreams are doing it....so consistency.

Reply:
>  It was moved into each file because, apparently, that is how all the downstreams are doing it....so consistency.

But it's completely unnecessary and shouldn't be done. All test projects (ones that have the Microsoft.NET.Test.Sdk installed) will automatically exclude themselves from producing a package.

If there is a scenario where there's a non-test project that also shouldn't be produce a package, then it makes more sense to mark that one project as not requiring it.

Reply:
I also would prefer not to see a condition on the property.